src/handlers/deletelink.ts

```diff
@@ -0,0 +1,51 @@
+import {
+	parseDeleteString,
+	constructSlackMessage,
+} from '../utils';
+
+/**
+ * Sends POST to link shortening service
+ *
+ * @param path – parsed path string
+ * @param url - parsed url string
+ */
+const deleteOldUrl = (path: string): Promise<Response> => {
+	const fetchUrl = `https://vhl.ink/${path}`;
+
+	const headers = new Headers({
+		'Content-Type': 'application/x-www-form-urlencoded',
+		'x-preshared-key': SECRET_KEY,
+	});
+
+	return fetch(`${fetchUrl}`, {
+		method: 'DELETE',
+		headers
+	});
+};
+
+/**
+ * Handles shortening links. Expects the text to look like:
+ * `/shorten <path> <url>`, and result in `vhl.ink/path`
+ */
+export default async (request: Request, text: string) => {
+	try {
+		// https://api.slack.com/tutorials/slash-block-kit is more updated than the Cloudflare tutorial
+		const params: { [key: string]: string } | undefined = parseDeleteString(
+			text,
+		)?.groups;
+		const deleteKW: string | undefined = params?.delete;
+		const path: string | undefined = params?.path;
+		const key: string | undefined = params?.key;
+
+		if (deleteKW && path && key && deleteKW == "delete" && key == SECRET_KEY) {
```

Comment:
is it possible to consistently use ===

Reply:
also if `deleteKW` is undefined then wouldn't `deleteKW === "delete"` automatically be false in that case? all that is to say do we need that extra first check